sanic/app.py

```diff
@@ -50,13 +50,18 @@ def __init__(self, name=None, router=None, error_handler=None):
         self.debug = None
         self.sock = None
         self.listeners = defaultdict(list)
+        self.is_running = False
 
         # Register alternative method names
         self.go_fast = self.run
 
     @property
     def loop(self):
         """Synonymous with asyncio.get_event_loop()."""
+        if not self.is_running:
```

Comment:
Good point, wouldn't `asyncio.get_event_loop()` still return that loop though? In any case, this message is a little unclear: https://github.com/channelcat/sanic/pull/459/files#diff-c4444a35dbd5e37ee480b0e8888e0880R64

I would phrase that (for the current situation): "Loop can only be retrieved after the app has started running. Not supported with run_async"